app/views/api/vacancies/index.json.jbuilder

```diff
@@ -1,3 +1,19 @@
-json.vacancies @vacancies.decorated_collection do |vacancy|
+json.info do
+  json.title "GOV UK - #{I18n.t('app.title')}"
+  json.description I18n.t('app.description')
+  json.termsOfService I18n.t('service.url')
+  json.contact do
+    json.name "#{I18n.t('app.title')} API Support"
+    json.url 'tbc'
```

Comment:
I see there was service.url added to locale, was the intent to use that here?

Reply:
@tahb actually I was unsure of what to use here as this is meant to be a contact url and we don't have a contact page. I think we can remove this field since there isnt one, and just leave the email address.

Reply:
What about `mailto:teachingjobs@digital.education.gov.uk`?

Reply:
It specifically indicates that it should be a url. However it's not a required field. 

<img width="918" alt="screen shot 2018-11-06 at 13 33 53 1" src="https://user-images.githubusercontent.com/159200/48067647-aef80a00-e1c8-11e8-8bca-27d07efce384.png">

Reply:
So it does, URL rather than URI. I'll stop trying to be clever! If it's optional I'm in favour of removing it as it's unlikely it will ever be corrected from 'tbc' even if a contact page is eventually created.